data-raw/prepare_data_for_model.R

```diff
@@ -317,6 +317,33 @@ degree_days_sr <- list(biop_2008_2009 = degree_days_2008_2009_sr,
 
 usethis::use_data(degree_days_sr, overwrite = TRUE)
 
+# degree days for above dam that max out at 17 degrees
+# TODO: is 17 the correct threshold value. Double check. Explore how the threshold affects output
+
+## 2018 2019
+monthly_mean_temperature_2018_2019_abv_dam <- monthly_mean_temperature_2018_2019 |>
+  mutate(monthly_mean_temp_c = ifelse(monthly_mean_temp_c >= 13, 13, monthly_mean_temp_c))
+temperatures_2018_2019_abv_dam <- temperatures_2018_2019 |>
+  mutate(mean_daily_temp_C = ifelse(mean_daily_temp_C >= 13, 13, mean_daily_temp_C),
+         mean_daily_temp_F = ifelse(mean_daily_temp_F >= 62.6, 62.6, mean_daily_temp_F))
+
+degree_days_2018_2019_sr_abv_dam <- generate_degree_days(monthly_mean_temperature_2018_2019_abv_dam,
```

Comment:
`hec_version` should be "2018 & 2019 Hec5q" not "2008 & 2009 Hec5q" since we are doing it for 2018 2019 temps.